app/src/test/kotlin/io/eugenethedev/taigamobile/repositories/TasksRepositoryTest.kt

```diff
@@ -791,4 +790,27 @@ class TasksRepositoryTest : BaseRepositoryTest() {
             )
         }
     }
+
+    @Test
+    fun `test edit blocked`() = runBlocking {
+        val dataForTest = getCommonTasksExt(
+            listOf(
+                CommonTaskType.Epic,
+                CommonTaskType.Issue,
+                CommonTaskType.Task,
+                CommonTaskType.UserStory
+            )
+        )
+        val blockReason = "Reason"
+
+        dataForTest.forEach { (_, data) ->
```

Comment:
Why use brackets?

Reply:
It's called [Destructuring declarations](https://kotlinlang.org/docs/destructuring-declarations.html). It's really handy when you want to unpack object to a set of variables (e.g. when [iterating over map entries](https://kotlinlang.org/docs/destructuring-declarations.html#example-destructuring-declarations-and-maps))